docs/src/use/configure/migration-guide.md

```diff
@@ -0,0 +1,339 @@
+---
+title: Configuration Migration Guide
+eleventyNavigation:
+    key: migration guide
+    parent: configure
+    title: Configuration Migration Guide
+    order: 8
+---
+
+This guide provides an overview of how you can migrate your ESLint configuration file from the `.eslintrc` format to the new configuration file format, `eslint.config.js`.
+
+To learn more about the `eslint.config.js` file format, refer to [this blog post](https://eslint.org/blog/2022/08/new-config-system-part-2/).
+
+For reference information on these configuration formats, refer to the following documentation:
+
+* [`.eslintrc` configuration files](configuration-files)
+* [`eslint.config.js` configuration files](configuration-files-new)
+
+## Start Using `eslint.config.js`
+
+As of ESLint v9.0.0, the `eslint.config.js` file format is the default configuration file format. If you are using ESLint v9.0.0 or later, you can start using the `eslint.config.js` file format without any additional configuration.
+
+To use `eslint.config.js` with ESLint v8.x.x, place a `eslint.config.js` file in the root of your project **or** set an `ESLINT_USE_FLAT_CONFIG` environment variable to `true`.
```

Comment:
to reviewer: is there a specific version here where as long as `eslint.config.js` is present (even if there's an .eslintrc` file, the flat config is used by default?

Reply:
`eslint.config.js` lookup was introduced in [v8.23.0](https://github.com/eslint/eslint/releases/tag/v8.23.0). `ESLINT_USE_FLAT_CONFIG` came later in [v8.26.0](https://github.com/eslint/eslint/releases/tag/v8.26.0). Not sure if we need to go into that detail here, especially since we are assuming that ESLint 9 has been already released.

Reply:
thanks, refactoring to just 'v8'

```suggestion
To use `eslint.config.js` with ESLint v8, place a `eslint.config.js` file in the root of your project **or** set an `ESLINT_USE_FLAT_CONFIG` environment variable to `true`.
```